go.mod

```diff
@@ -28,16 +28,16 @@ require (
 	github.com/percona/percona-toolkit v3.2.1+incompatible
 	github.com/percona/pmm v0.0.0-20220310104949-dd33560bcdde
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.5.1
-	github.com/prometheus/common v0.10.0
+	github.com/prometheus/client_golang v1.11.1
```

Comment:
Any reason to downgrade it from `v1.12.1` to `v1.11.1` ?

Reply:
The log package got removed from prometheus so we downgraded to keep it - but there's no reason why we can't remove it and directly use logrus too, so I've updated the code now.